specification/communication/data-plane/CallingServer/preview/2022-04-07-preview/communicationservicescallingserver.json

```diff
@@ -891,6 +932,62 @@
         "callbackUri": {
           "description": "The callback uri.",
           "type": "string"
+        },
+        "mediaStreamingConfiguration": {
```

Comment:
Any reason to have this only on the incoming call but not also on making the outbound call?